mm-go-irckit/service.go

```diff
@@ -272,6 +272,20 @@ func searchUsers(u *User, toUser *User, args []string, service string) {
 	}
 }
 
+func getChannelName(u *User, channelID string) string {
```

Comment:
I don't really like that we have yet another mattermost specific getchannelname thing :)
This should ideally be done in `bridge/{mattermost,slack,mastodon}`
Or if too hard to fit in there it should at least be renamed here getMattermostChannelName

Reply:
Okay, will thanks. Thanks for your review.

Reply:
Had a look, looks to be doing different things. Renamed as suggested.